packages/@ama-sdk/core/src/fwk/api.helpers.spec.ts

```diff
@@ -0,0 +1,33 @@
+import { extractInfoFromUrl } from './api.helpers';
+
+describe('ExtractInfoFromUrl', () => {
```

Comment:
your forgot to test your supported Basic Auth URL

Reply:
I removed the factory function and the test too. I found another way to get the origin from the `headers` directly